src/ansible.coffee

```diff
@@ -35,3 +35,5 @@ module.exports = (robot) ->
     command = msg.match[1]
 
     runAnsiblePlaybook msg, command
+
+  robot.on 'ansible:runPlaybook', runAnsiblePlaybook
```

Comment:
Hum, is it as simple as that? `runAnsiblePlaybook` takes arguments. Does the `on` method pass arguments to it?